solution/zenkoversion.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: zenko.io/v1alpha1
+kind: ZenkoVersion
+metadata:
+  name: ${VERSION_FULL}
+  namespace: SOLUTION_ENV
```

Comment:
minor: this can probably be removed and use the '--namespace' option

Reply:
created follow up story to also add docs https://scality.atlassian.net/browse/ZENKO-3089